ci.sh

```diff
@@ -4,9 +4,12 @@
 set -e
 
 # Ensure no compile errors in all projects
-find . -name *.csproj -exec dotnet build {} \;
+dotnet restore
+dotnet build --no-restore
 
 # Execute Unit tests
 cd tests
-dotnet restore
-dotnet test
+dotnet test --no-restore --no-build
+if [[ $? != 0 ]]; then
```

Comment:
already set -e

Reply:
Yeah, I was wondering about that - it's supposed to stop on error (if you have set -e) right? But if I take that out then the build doesn't fail if tests fail :-/

Reply:
I'm no bash expert so it's entirely possible I've got it wrong though.

Reply:
I can't remember where, but I remember reading something a while back that said "set -e" is unreliable in some cases and you're better off using "trap" but again, I could be wrong :)